test/suites/profiling_gcc_10+.bash

```diff
@@ -0,0 +1,111 @@
+# Remove header, including a volatile timestamp, from a .gcno file.
```

Comment:
The comment does not seem to describe the function?

Reply:
Copy-paste leftover, deleted.